lib/install/opsuser/opsuser.go

```diff
@@ -98,12 +101,27 @@ func (mgr *RBACManager) SetupDCReadOnlyPermissions(ctx context.Context) (*rbac.R
 	if err != nil {
 		return nil, err
 	}
+
 	// If administrator, skip setting the root permissions
 	if res {
 		log.Warnf("Cannot perform ops-user Role/Permissions initialization. The current ops-user (%s) has administrative privileges.", am.Principal)
 		log.Warnf("This occurs when \"%s\" is a member of the \"Administrators\" group or has been granted \"Admin\" role to any of the resources in the system.", am.Principal)
 		return nil, errors.Errorf("Cannot grant ops-user permissions as %s has administrative privileges", am.Principal)
 	}
+
+	dcRef := mgr.session.Datacenter.Reference()
+	hasPrivs, err := am.ReadOnlyOnRef(ctx, dcRef)
```

Comment:
The reason for granting read-only permissions to the ops-user is to ensure they are allowed to log in so that we can verify the username and password - @lcastellano found that if, at a minimum, ReadOnly is not granted at the datacenter level then the user cannot authenticate.

Please ensure that the changes to this path are tested vs a VC where the user does _not_ have these privileges and cannot log in before this code is run.

Really this logic granting read-only should be present as a remediate-and-retry after failing to validate ops-user credentials.

Reply:
> Please ensure that the changes to this path are tested vs a VC where the user does not have these privileges and cannot log in before this code is run.

Verified this works.

Reply:
Opened https://github.com/vmware/vic/issues/7606 for follow-on work per our conversation.